vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -1,3 +1,177 @@
+
+#### Command Details
+`ent bundle build`: Constructs the project files based on the entando.json provided. It executes according to the type of stack your components are built with. For instance, npm build will be performed for a React MFE. A log file is generated for each component inside the .entando/logs directory for your project.
+
+## Run	
+| Command| Descriptions
+|:--|:--
+|`ent bundle run [component-name]` | Locally run your single component
+|`ent bundle run [mfe-1] [ms-2]...`| Locally run all the components passed as arguments 
+|`ent bundle run --all`| Locally run all the components in the bundle
+|`ent bundle run --all-ms`| Locally run the MSs in the bundle
+|`ent bundle run --all-mfe`| Locally run the MFEs in the bundle
+
+#### Command Details
+`ent bundle run`: Executes processes dependent on the component type and stack. For example, MVN spring:boot will execute for a Sping Boot microservice. All the components in the bundle will run in parallel, with a log file generated for each inside the .entando/logs directory.
+
+## Package
+| Command| Descriptions
+|:--|:--
+|`ent bundle pack`|Generate the bundle artifacts, and the bundle Docker image and MS images 
+|`ent bundle pack --org [organization]`|Generate the bundle artifacts, passing the organization name to the Docker Hub|
+|`ent bundle pack --file [my-dockerfile]`|Use a custom Dockerfile for your bundle |
+
+#### Command Details
+* `ent bundle pack`: For the MFEs and MSs, the generated artifacts are located inside each component's folders. Their format depends on the component type.  For instance, a React micro frontend may result in HTML, JavaScript and CSS files.  
+
+When artifact generation is complete, microservices Docker images are built, executing the docker build command using the Dockerfile located in each microservice folder. If the Dockerfile is missing, the pack command exits with failure.
+
+** Should we include info about where yaml descriptors are located, how they are produced from entando.json, pom.xml or package.json, or how to see the outputs for the MFE in the DEBUG mode
+
+## Publish
+| Command| Descriptions
+|:--|:--
+|ent bundle publish| Publish your Docker images to index.docker.io (default)
+|ent bundle publish --org [organization]|Publish Docker images to the Docker registry, with a specified organization |
+|ent bundle publish --registry [registry]| Publish the Docker images to your Docker registry|
+
+#### Command Details
+* `ent bundle publish --org`: If the organization name has changed, the Docker tag command can be used to create new image tags for the new name to be stored in .entando/config.json.
+
+   docker tag OLD-ORG-NAME/YOUR-IMAGE:tag NEW-ORG-NAME/YOUR-IMAGE:tag
+
+* `ent bundle publish --registry`
+   1. A specific Docker registry can be used but it must be authenticated by using the --config flag on the login command.
+     ```
+     docker --config path/to/.docker login YOUR-REGISTRY
```

Comment:
This is an internal implementation detail, the user should never execute `docker` command by himself/herself. `ent bundle` executes this commands internally.